app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
+				"shelter", "hospital", "police", "community_centre",
```

Comment:
hospital is a spam candidate, too

Reply:
The goal of this tag is explicit tagging by a person who is familiar with this topic and did a survey. Sure, most hospitals, clinic, doctors and pharmacies are accessible by wheelchair users. But, where do you see the differences between hospital and clinic?